worker/src/RTC/SvcConsumer.cpp

```diff
@@ -109,24 +109,21 @@ namespace RTC
 		// Call the parent method.
 		auto base = RTC::Consumer::FillBuffer(builder);
 		// Add rtpStream.
-		auto rtpStream = this->rtpStream->FillBuffer(builder);
+		std::vector<flatbuffers::Offset<FBS::RtpStream::Dump>> rtpStreams;
+		rtpStreams.emplace_back(this->rtpStream->FillBuffer(builder));
 
-		auto svcConsumerDump = FBS::Consumer::CreateSvcConsumerDump(
+		auto dump = FBS::Consumer::CreateConsumerDumpDirect(
 		  builder,
 		  base,
-		  rtpStream,
+		  &rtpStreams,
```

Comment:
I like using `std::addressof(rtpStreams)` instead of C style `&rtpStreams`, but it's not critical.